src/main/java/com/kruize/metrics/ContainerMetrics.java

```diff
@@ -115,8 +115,6 @@ public double getCurrentCpuLimit()
     }
 
     @Override
-    public double getCurrentRssLimit()
-    {
-        return currentRecommendations.getRssLimit();
+    public double getCurrentRssLimit() { return currentRecommendations.getRssLimit();
```

Comment:
If you are moving to a single line, please move the ending bracket as well.

Reply:
Okay.